### ISSUE TITLE
maxContentSizeMultiplier not working on Text

### ISSUE THREAD

**Issue by jessamp** [Microsoft/reactxp#1052]
I tried to set the maxContentSizeMultiplier prop on a Text component, but it looks like it's not changing anything. React native has something similar to this called maxFontSizeMultiplier. Is this supposed to have the same behavior?

**Comment by erictraut** [comment]
Yes, it looks like this functionality finally made it into RN, but it's under a different name than the one the Skype team initially used. See [#783](https://github.com/Microsoft/reactxp/issues/783) for more details. We should change the ReactXP implementation accordingly. I'd prefer to stick with the "maxContentSizeMultiplier" name in the ReactXP interface to avoid a breaking change for ReactXP.

**Comment by jessamp** [comment]
Yeah we've used those functions :). 
When will this be available for public use? Or would it be helpful to submit a PR?

**Comment by fbartho** [comment]
@erictraut / @jessamp -- now that TypeScript supports the [`@deprecated`](https://github.com/microsoft/TypeScript/pull/38523) feature (and VSCode Displays it!)

I think we could start providing the new API, and cross-map the old values to the new ones. What do you think?

**Comment by erictraut** [comment]
Yeah, good suggestion.
